### ISSUE TITLE
Firefox support for web client

### ISSUE THREAD

**Issue by matzayonc** [kixelated/web-transport-rs#15]
When I open the web client using Firefox I get a not-too-informative `WebTransport connection rejected` error. I have the newest version, which is green on [caniuse](https://caniuse.com/webtransport).
On Chrome it works just fine.

I would be open to helping out, given some lead.

**Comment by kixelated** [comment]
Going to finally debug this tomorrow, heh.

**Comment by billyb2** [comment]
@matzayonc if you're using self signed certificates, I believe that Firefox doesn't support those:
https://superuser.com/questions/1780337/let-firefox-accept-a-website-with-self-signed-certificate

**Comment by kixelated** [comment]
> [@matzayonc](https://github.com/matzayonc) if you're using self signed certificates, I believe that Firefox doesn't support those: https://superuser.com/questions/1780337/let-firefox-accept-a-website-with-self-signed-certificate

Yeah so local development does use a local certificate supplied via: https://developer.mozilla.org/en-US/docs/Web/API/WebTransport/WebTransport#servercertificatehashes

I just tested and Firefox does work on quic.video (LetsEncrypt) so this is definitely a cert issue.

**Comment by matzayonc** [comment]
Cool, I will try again on the first occasion. Thanks for replying and closing
